spec/controllers/applications_controller_spec.rb

```diff
@@ -126,11 +126,13 @@
           id: application.id,
           user:
           {
+            email_for_google: :email_for_google,
```

Comment:
If you want to refer to the variable you are defining on line 135, this should be `email_for_google` (without the `:`). In this form you are passing in a symbol `:email_for_google` (which is totally ok and works fine for a test, but then you don't need to define the variable on line 135, since you are not using it anywhere).

Reply:
Thank you! New to Ruby and had not figured out what symbols were / what their syntax looked like yet. Fixed.